pkg/kube/util/ctxlog/context.go

```diff
@@ -34,6 +34,11 @@ func NewContextWithRecorder(ctx context.Context, name string, recorder record.Ev
 	return context.WithValue(ctx, ctxLoggerKey, log)
 }
 
+func ExtractLoggerWithSkip(ctx context.Context, skip int) *zap.SugaredLogger {
```

Comment:
We should give a comment if we follow go lint.

Reply:
Yes, even worse, the function was ignoring the skip parameter....